migrations/2021_02_12_134612_create_webhook_notification_messages_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateWebhookNotificationMessagesTable extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('webhook_notification_messages', function (Blueprint $table) {
+            $table->uuid('id');
+            $table->unsignedBigInteger('notifiable_id');
+            $table->string('notifiable_type');
```

Comment:
```suggestion
            $table->morphs('notifiable');
```
will also create an index on both columns

Reply:
Done.

Reply:
We only need one (composite) index, which is exactly what the `morphs` method does.